src/View/Board.elm

```diff
@@ -32,10 +32,10 @@ view (Config { toSvg, toMsg }) data =
         container =
             svg
                 [ viewBox "-10 -10 20 20"
-                , style "flex-basis: 70%"
+                , style "flex-basis: 60%"
                 ]
 
         makeNode datum =
             g [ onClick (toMsg datum) ] [ toSvg datum ]
     in
-        data |> List.map makeNode |> container
+    data |> List.map makeNode |> container
```

Comment:
Same formatting question here. Maybe we're running different versions of `elm-format`?